test/integration/scheduled-action-integration.ts

```diff
@@ -0,0 +1,267 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { expect } from 'chai'
+import { before, describe, test } from 'mocha'
+import { Space } from '../../lib/entities/space'
+import { TestDefaults } from '../defaults'
+import { getDefaultSpace, getPlainClient } from '../helpers'
+import { makeLink } from '../utils'
+
+describe('Scheduled actions api', async function () {
+  let testSpace: Space
+
+  before(async () => {
+    testSpace = await getDefaultSpace()
+  })
+
+  describe('Read', () => {
+    test('Get Scheduled action', async () => {
+      const createdAction = await testSpace.createScheduledAction({
+        entity: makeLink('Entry', TestDefaults.entry.testEntryId),
+        action: 'publish',
+        scheduledFor: {
+          datetime: new Date().toISOString(),
+        },
+      })
+
+      const fetchedActions = await testSpace.getScheduledActions({
+        'environment.sys.id': createdAction.environment.sys.id,
+        'sys.id[in]': createdAction.sys.id,
+      })
+
+      expect(fetchedActions.total).to.eql(1)
+      const fetchedAction = fetchedActions.items[0]
+      expect(fetchedAction.sys.id).to.eql(createdAction.sys.id)
+      expect(fetchedAction.action).to.eql(createdAction.action)
+      expect(fetchedAction.entity).to.eql(createdAction.entity)
+
+      // cleanup
+      await fetchedAction.delete()
```

Comment:
Do we need to put clean up in a `finally {}` block?

Reply:
I guess in case of an error, scheduled action wouldn't be created, so there will be nothing to cleanup, right?

Reply:
depends on where it failed (could also fail on fetch)